src/saturn_engine/utils/flask.py

```diff
@@ -0,0 +1,17 @@
+from typing import Generic
+from typing import TypeVar
+from typing import cast
+
+import flask
+
+from .options import asdict
+
+T = TypeVar("T")
+
+
+class Json(Generic[T]):
+    pass
+
+
+def jsonify(item: T) -> Json[T]:
```

Comment:
Only serves to make sexy signatures?

Reply:
And ensure we serialize from the API object. If someone were to change the API object, the client and server should scream.

Reply:
Yeah cool